strings/ion_laws.json

```diff
@@ -1,5 +1,132 @@
 {
-    "ionabstract": [
+    "ionspecies": [
+        "LIFEFORMS WITHOUT TAILS",
+		"LIFEFORMS WITHOUT WINGS",
+		"INORGANIC LIFEFORMS",
+		"LIFEFORMS WITH TAILS",
+		"LIFEFORMS WITH WINGS",
+		"LIFEFORMS HOLDING A SOURCE OF LIGHT",
+		"LIFEFORMS HOLDING A TOOLBOX",
+		"LIFEFORMS HOLDING A FIRE EXTINGUISHER",
+		"LIFEFORMS WITH AN OBSCURED IDENTITY",
+		"LIFEFORMS THAT HAVE FARTED RECENTLY",
+		"LIFEFORMS THAT ARE NOT WEARING A BAG",
+		"LIFEFORMS THAT ARE WEARING A PAPER HAT",
+		"LIFEFORMS THAT ARE WEARING A CLOAK",
+		"LIFEFORMS THAT ARE NOT WEARING SHOES",
+		"LIFEFORMS THAT HAVE RECENTLY SAID NYA"
+    ],
+    "ionfragment": [
+        "VISIBLY CARRIES A GUN",
+		"HAS BLOOD ON THEIR CLOTHING",
+		"BREAKS SPACE LAW",
+		"HAS ANY DISEASE",
+		"IS WEARING A HARDSUIT"
+    ],
+    "iondepartment": [
+        "SECURITY",
+		"ENGINEERING",
+		"THE MEDBAY",
+		"RESEARCH AND DEVELOPMENT",
+		"MAINTENANCE",
+		"BOTANY",
+		"CARGO",
+		"SPACE"
+    ],
+    "ionkeyword": [
+        "VALID",
+		"GAMER",
+		"SEX",
+		"CUM",
+		"SHITSEC",
+		"ANTAG",
+		"SHUTTLE",
+		"CATGIRL",
+		"MALF",
+		"ANY CURSE WORD"
+    ],
+    "ionaction": [
+		"GETTING DRUNK",
+		"RUNNING",
+        "BEING HANDCUFFED",
+		"EATING",
+		"SURGERY",
+		"SQUEAKING OR HONKING"
+    ],
+    "ionequipment": [
+        "APCS",
+		"GRAVITY GENERATORS",
+		"TELECOMMUNICATIONS",
+		"SOURCES OF LIGHT",
+		"COMPUTER CONSOLES"
```

Comment:
again probably potential for a lot more items in here, 5 seems like a small amount

Reply:
Even the old list has a lot more you could steal, airlocks, windows, windoors, buttons, air vents, air scrubbers, the experimentor, shuttles, crates etc.

Reply:
Yet again, the context of the law that pulls from this list matters a lot. Almost none of what you listed works for this variable.

5 is definitely a small amount though, and I would like to have more.